src/server/response.cpp

```diff
@@ -124,6 +125,14 @@ void Response::introduce_taskbar()
     m_content,
     "<body[^>]*>",
     taskbar_part);
+
+  if ( m_blockExternalLinks ) {
+    const std::string capture_external_part = getResource("templates/block_external.js");
+    m_content = appendToFirstOccurence(
+      m_content,
+      "block external links\n",
+      capture_external_part);
+  }
```

Comment:
I would move this code in another function than introduce taskbar.

- This is not related to taskbar.
- Now, how it is, if taskbar is deactivated, you don't include `block_external.js`. So user cannot block external links without a taskbar.